src/pantry.js

```diff
@@ -40,34 +40,69 @@ class Pantry {
     return (ingredientsNeeded.length > 0) ? false : true;
   }
 
-
   determineAdditionalNeededIngredients(chosenRecipe) {
     let ingredientsNeeded = this.compareRecipeToPantryIngredients(chosenRecipe);
     let neededQuantity = ingredientsNeeded.map(ingredient => {
-      return `${ingredient.name} ${ingredient.qty} ${ingredient.unit}`
+      return `<li>${ingredient.qty.toFixed(2)} ${ingredient.unit} - ${ingredient.name}</li>`
     })
     return neededQuantity
   }
 
   calculateCostOfAdditionalIngredients(chosenRecipe) {
     let ingredientsNeeded = this.compareRecipeToPantryIngredients(chosenRecipe);
-    let additionalIngredientCost = ingredientsNeeded.map(ingredient => {
-      return ingredient.name + ' $' + (ingredient.qty * ingredient.cost)/100
-    })
     let totalCost = ingredientsNeeded.reduce((cost, cur) => {
       cost += (cur.qty * cur.cost)/100
       return cost
     }, 0).toFixed(2)
-    additionalIngredientCost.push('total $' + totalCost)
-    return additionalIngredientCost
+    return totalCost;
   }
 
-  updatePantryContent() {
-
+  updatePantryContent(user, chosenRecipe) {
```

Comment:
Looks like the network request is working great! We will have to remember to check to see if we need to update the users global data since we are only updating the database with these functions.